client/App.tsx

```diff
@@ -1,7 +1,26 @@
 import React from "react";
+import { Route, Routes} from "react-router-dom";
+import ZoopsList from './components/ZoopList';
+import LoginPage from "./components/LoginPage";
+import Zoop from "./components/Zoop";
+import RegisterPage from "./components/RegisterPage"
+import MePage from "./components/MePage";
+import HomePage from "./components/HomePage";
 
 const App: React.FC = () => {
-  return <h1>Zoop is cool</h1>;
+  return (
+    <>
+      <Routes>
+        <Route path="/" element={<HomePage />}/>
+        <Route path="/zoops" element={<ZoopsList />}>
+          <Route path=":id" element={<Zoop />} />
+        </Route>
```

Comment:
You shouldn't need to do sub-routing like this? I am pretty sure that React Router v6 supports exact matches. But if you're encountering issues, you could put /zoops/:id first in the list of routes instead